static.webpack.config.common.js

```diff
@@ -33,15 +32,16 @@ class MergePortalConfigsPlugin {
         const output = {}
         const portalDirectory = 'portals'
         const directories = fs.readdirSync(portalDirectory, { withFileTypes: true })
-          .filter((dirent) => dirent.isDirectory())
+          .filter((dirent) => dirent.isDirectory() || !dirent.name.startsWith('.'))
 
         directories.forEach((directory) => {
           const { name } = directory
-          const contents = JSON.parse(fs.readFileSync(`${portalDirectory}/${name}/config.json`, 'utf8'))
-
-          output[name] = {
-            ...contents,
-            portalId: name
+          if (name !== 'mergedPortalConfigs.json') {
```

Comment:
I thought the filter of `dirent.isDirectory()` would keep the merged file from being picked up in the directories array, did you see it coming through?

Reply:
oh, it should be `&&`, not `||`